File: result/translate/starchat/java2python/python/ARRAY_RANGE_QUERIES_ELEMENTS_FREQUENCY_VALUE.py
def f_gold(start, end, arr):
        mp = {}
        for i in range(start, end + 1):
            mp[arr[i]] = mp.get(arr[i], 0) + 1
        count = 0
        for entry in mp.items():
            if entry[1] == max(mp.values()):
                count += 1
        return count

File: result/translate/starchat/java2cpp/c++/ARRAY_RANGE_QUERIES_ELEMENTS_FREQUENCY_VALUE.cpp
int f_gold ( int start, int end, int arr [ ] ) {
    unordered_map<int, int> mp;
    for (int i = start; i <= end; i++) mp[arr[i]]++;
    int count = 0;
    for (auto &x : mp) if (x.second == 1) count++;
    return count;
}